Products/ZenModel/UserSettings.py

```diff
@@ -449,6 +449,28 @@ def manage_deleteUsers(self, userids=(), REQUEST=None):
                 audit('UI.User.Delete', username=userid)
             return self.callZenScreen(REQUEST)
 
+    security.declareProtected(ZEN_MANAGE_DMD, 'manage_unlockUsers')
+    @validate_csrf_token
+    def manage_unlockUsers(self, userids=(), REQUEST=None):
+        """Unlocked a list of zenoss users from the system which are locked
+            due to numerous bad login attempts.
```

Comment:
No need to indent this line (the line starting with 'due').  It should be lined up with the first '"', e.g.
```
    """Unlocked a list of zenoss users from the system which are locked
    due to numerous bad login attempts.
    """
```